doc/dev/decisionrecords/Codestyle.md

```diff
@@ -144,41 +145,41 @@ context. Please avoid including trivial Javadoc or the empty Javadoc stubs added
     - Contract of the method
         - Input domain for which the logic is designed
         - Range of outputs produced from valid inputs
-        - Is behavior undefined or will fail when conditions are not met
-        - Are null values allowed as inputs
-        - Will null values occur as outputs (what do they mean)
+        - Is behavior undefined or will the method fail when conditions are not met?
+        - Are null values allowed as inputs?
+        - Will null values occur as outputs (and what do they mean)?
     - Invariants that hold if the preconditions are met
     - Concurrency
-        - Is method thread-safe
+        - Is the method thread-safe?
         - Usage constraints for multi-threaded use
 - On classes:
     - Initialization and teardown process
-    - Can instance be reused for multiple operations, or should it be discarded
-    - Is it immutable or should anything be treated as immutable
-    - Is it a utility class of static methods that should not be instantiated
+    - Can an instance be reused for multiple operations, or should it be discarded?
+    - Is it immutable, or should anything be treated as immutable?
+    - Is it a utility class of static methods that should not be instantiated?
 
 ### Annotations
 
 - On methods:
-    - Method should be marked as `@Nullable` if they can return null values
+    - Method should be marked as `@Nullable` if they can return null values.
     - Method parameters should be marked as `@Nullable` if they can take null values.
 - On fields:
     - Fields should be marked as `@Nullable` if they are nullable.
 
-Use of `@Nonnull` annotation is not allowed. It should be assumed methods/parameters/fields
-are non-null if they are not marked as `@Nullable`. However, there are places where the
-`@Nullable` annotation is missing even if it should have been used. Those can be updated
-to use the `@Nullable` annotation.
+Use of `@NonNull` annotation is not allowed. It should be assumed methods/parameters/fields are
```

Comment:
```suggestion
Use of `@Nonnull` annotation is not allowed. It should be assumed methods/parameters/fields are
```

Reply:
The javax annotation is `@Nonnull`, Lombok's annotation is `@NonNull` (although we shouldn't use that one either).

Reply:
I see, thank you. Reverted